sixtyfps_runtime/rendering_backends/gl/lib.rs

```diff
@@ -43,7 +43,7 @@ mod fonts;
 
 type CanvasRc = Rc<RefCell<femtovg::Canvas<femtovg::renderer::OpenGl>>>;
 
-const KAPPA90: f32 = 0.5522847493;
+const KAPPA90: f32 = 0.55228;
```

Comment:
on the other hand, it doesn't hurt to have it, does it?

Reply:
Not really, it just makes people assume that the results will be more precise than they really are.